# Define the goal
string[] total_missions  
---
# Define the result
uint32 missions_done
uint32 missions_failed
---
# Define a feedback message
string current_mission
int32 mission_state # 0-working, 1-done, -1-failed
